.gitmodules

```diff
@@ -8,7 +8,7 @@
 	shallow = true
 [submodule "qt/bundle/PyOxidizer"]
 	path = qt/bundle/PyOxidizer
-	url = https://github.com/ankitects/PyOxidizer.git
+	url = https://github.com/indygreg/PyOxidizer.git
```

Comment:
I pinned the submodule to the [latest release tag](https://github.com/indygreg/PyOxidizer/releases/tag/pyoxidizer%2F0.24.0).

Reply:
It is needed - we have some custom patches necessary to build the bundle, and they'd need to be rebased over the latest code.